core/src/test/kotlin/repository/UserRepositoryIT.kt

```diff
@@ -0,0 +1,176 @@
+package repository
+
+import org.bson.types.ObjectId
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import reactor.test.StepVerifier
+import rys.ajaxpetproject.model.MongoUser
+import rys.ajaxpetproject.repository.UserRepository
+
+@DbIntegrationTest
+class UserRepositoryIT {
+    @Autowired
+    private lateinit var userRepository: UserRepository
+
+    @BeforeEach
+    fun init() {
+        userRepository.deleteAll().block()
+    }
+
+    @Test
+    fun `should return user when findById() success`() {
+        //GIVEN
+        val id = ObjectId("652f9203deb9044f309a7dd9")
+        val user = MongoUser(
+            id = id, userName = "Successful_findUserById - ${System.nanoTime()}",
+            password = "password"
+        )
+        userRepository.save(user).block()
+
+        //WHEN //THEN
+        StepVerifier.create(userRepository.findById(id))
+            .expectSubscription()
+            .expectNext(user)
+            .verifyComplete()
+    }
+
+    @Test
+    fun `should return user when findByName() success`() {
+        //GIVEN
+        val givenName = "Successful_findUserById - ${System.nanoTime()}"
+        val user = MongoUser(
+            id = ObjectId("652f9203deb9044f309a7dd9"),
```

Comment:
Don't hardcode ID in your test. I bet earlier or later this test will accidentally fail
```suggestion
```